vet.sh

```diff
@@ -157,7 +157,22 @@ grpc.WithTimeout
 http.CloseNotifier
 info.SecurityVersion
 resolver.Backend
-resolver.GRPCLB' "${SC_OUT}"
+resolver.GRPCLB
+proto.RegisterType is deprecated
+proto.RegisterEnum is deprecated
+proto.RegisterFile is deprecated
+proto.FileDescriptor is deprecated
+proto.MessageType is deprecated
+proto.RegisteredExtensions is deprecated
+extDesc.Filename is deprecated
+proto.RegisteredExtensions
```

Comment:
What's the full text of this line?

Reply:
reflection/serverreflection.go:244:24: proto.RegisteredExtensions is deprecated: Use protoregistry.GlobalTypes.RangeExtensionsByMessage instead.  (SA1019)

I shortened all of these but didn't mean to exclude "is deprecated here".  Fixed.